### PR TITLE
Fix issue #3

### PR DESCRIPTION
**Pull request by prempalsingh**
Now even private methods with @AfterPermissionGranted annotation will be executed.

### CODE REVIEW

**Comment by prempalsingh** [comment]
I signed it!

File: easypermissions/src/main/java/pub/devrel/easypermissions/EasyPermissions.java [hunk 1]
```diff
@@ -184,6 +184,8 @@ private static void runAnnotatedMethods(Activity activity, int requestCode) {
                     }
 
                     try {
+                        if(!method.isAccessible())
+                            method.setAccessible(true);
```
Comment (samtstern) [inline easypermissions/src/main/java/pub/devrel/easypermissions/EasyPermissions.java]: To keep style consistent, please add brackets to if statement.

**Comment by samtstern** [comment]
This change looks great, just one comment above.

**Comment by prempalsingh** [comment]
Should I amend the previous commit or make another commit?

**Comment by samtstern** [comment]
Whichever you prefer.

**Comment by prempalsingh** [comment]
Please check now.

**Comment by samtstern** [comment]
Thanks!  Merging.

**Pull request merged**